src/plugins/utils.ts

```diff
@@ -0,0 +1,8 @@
+export const insertPluginsToArray = <T>(pluginsArray: any[], array: T[]): T[] => {
```

Comment:
`pluginsArray` here should be either `unknown[]` or `T[]` (depending how strict we'd like to be)